Lib/mimetypes.py

```diff
@@ -439,6 +439,7 @@ def _default_mime_types():
         '.jpeg'   : 'image/jpeg',
         '.jpg'    : 'image/jpeg',
         '.js'     : 'application/javascript',
+        '.json'   : 'application/json',
```

Comment:
It's not totally out of context : files with extension .json are obvious candidates for compression, this is why I have included this minor change in mimetypes, thinking that it doesn't deserve a specific PR. But if you think it's better to open another one, I will.